client/components/LearnerViewDirective.js

```diff
@@ -446,10 +456,17 @@ tie.directive('learnerView', [function() {
           CodeStorageService, SECONDS_TO_MILLISECONDS, DEFAULT_AUTOSAVE_SECONDS,
           DISPLAY_AUTOSAVE_TEXT_SECONDS) {
         var DURATION_MSEC_WAIT_FOR_SCROLL = 20;
+        var ALLOWED_QUESTION_SET_IDS = ['strings', 'other', 'all'];
         var language = LANGUAGE_PYTHON;
         // TODO(sll): Generalize this to dynamically select a question set
         // based on user input.
         var questionSetId = 'strings';
+        var questionSetIds = [
```

Comment:
Not sure you need the "var" variable -- you can just define $scope.questionSetIds directly.

That said, probably better not to hardcode two things with the same data. Can $scope.questionSetIds be derived from ALLOWED_QUESTION_SET_IDS somehow?